examples/comment/reply_to_media_comments.py

```diff
@@ -0,0 +1,74 @@
+# coding=utf-8
+"""
+    instabot example
+
+    Workflow:
+        If media is commented, reply to comments if you didn't reply yet to that user.
+"""
+
+import argparse
+import os
+import sys
+from tqdm import tqdm
+
+sys.path.append(os.path.join(sys.path[0], '../../'))
+from instabot import Bot
+
+parser = argparse.ArgumentParser(add_help=True)
+parser.add_argument('-u', type=str, help="username")
+parser.add_argument('-p', type=str, help="password")
+parser.add_argument('-proxy', type=str, help="proxy")
+parser.add_argument('-comments_file', type=str, help="comments_file")
```

Comment:
If you want to set this argument as required
```suggestion
parser.add_argument('-comments_file', type=str, help="comments_file", required=True)
```
More:
https://docs.python.org/3/library/argparse.html#required